examples/cpp/core/dll/using_dll.cpp

```diff
@@ -59,6 +59,9 @@ int main()
     bond::RuntimeSchema schema = bond::GetRuntimeSchema<MyStruct>();
 
     std::cout << schema.GetSchema().structs[schema.GetSchema().root.struct_def].fields[0].metadata.name << std::endl;
```

Comment:
Consider, in a separate commit, adding a test that accesses the metadata of a generic struct.